### PR TITLE
Support for NetBox v3.4

### LINKED ISSUE #74: [Feature]: Netbox 3.4 Top Level Navigation
### NetBox version

v3.4.0

### Feature type

Add a function

### Proposed functionality

netbox-community/netbox#9071 added support for plugins to insert top level navigation.

### Use case

Move to top level navigation

### External dependencies

v3.4.x

### LINKED ISSUE #95: [Feature]: NetBox 3.4 Support
### NetBox version

v3.4.0

### Feature type

Add a function

### Proposed functionality

NetBox 3.4 Support

### Use case

NetBox 3.4 Support

### External dependencies

_No response_

### LINKED ISSUE #96: [Feature]: Move cards to a view tab
### NetBox version

v3.4.0

### Feature type

Add a function

### Proposed functionality

It would be good to move cards when it makes sense to a new view tab given the new function of NetBox 3.4

### Use case

Many of the cards for ACLs like interfaces or devices get lost in the shuffle the core NetBox cards.

### External dependencies

NetBox 3.4

### PR DESCRIPTION
**Pull request by abhi1693**
# Pull Request

## Related Issue

Closes #95 
Closes #96 
Closes #74 

## New Behavior

...

## Contrast to Current Behavior

...

## Discussion: Benefits and Drawbacks

...

## Changes to the Documentation

...

## Proposed Release Note Entry

...

## Double Check

* [x] I have explained my PR according to the information in the comments
 or in a linked issue.
* [ ] My PR targets the `dev` branch.

### CODE REVIEW

**Comment by ryanmerolle** [comment]
@abhi1693 would you mind resolving the conflicts?  Then I will PR some additional docker tweaks (remove initializers in the repo in favor of another approach) and some other items before I do fuller pre-release tests.

**Comment by ryanmerolle** [comment]
All in all, everything looks good to me at first review.

File: .devcontainer/Dockerfile-plugin_dev [hunk 1]
```diff
@@ -1,8 +1,9 @@
-ARG NETBOX_VARIANT=v3.3
+ARG NETBOX_VARIANT=v3.4
 
 FROM netboxcommunity/netbox:${NETBOX_VARIANT}
 
-ARG NETBOX_INITIALIZERS_VARIANT=3.3.*
+
```
Comment (abhi1693) [inline .devcontainer/Dockerfile-plugin_dev]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -37,10 +37,14 @@ See the [CONTRIBUTING](CONTRIBUTING.md) for more information.
 Each Plugin Version listed below has been tested with its corresponding NetBox Version.
 
 | NetBox Version | Plugin Version |
-|----------------|----------------|
-|       3.2      |      1.0.1     |
-|       3.3      |      1.1.1     |
-|       3.4      |  1.2.0(coming) |
+
```
Comment (abhi1693) [inline README.md]:
```suggestion
```

File: netbox_acls/__init__.py [hunk 3]
```diff
@@ -17,9 +17,9 @@ class NetBoxACLsConfig(PluginConfig):
     version = __version__
     description = "Manage simple ACLs in NetBox"
     base_url = "access-lists"
-    min_version = "3.3.0"
-    max_version = "3.3.99"
-    # default_settings = {}
+
```
Comment (abhi1693) [inline netbox_acls/__init__.py]:
```suggestion
```

File: setup.py [hunk 4]
```diff
@@ -23,6 +23,8 @@ def get_version(rel_path):
 
 setup(
     name="netbox-acls",
+
```
Comment (abhi1693) [inline setup.py]:
```suggestion
```

**Comment by abhi1693** [comment]
@ryanmerolle While you do testing, make sure to grab new screenshots for Readme as there are changes in the UI

**Pull request merged**